README.md

```diff
@@ -8,157 +8,155 @@
-This reduces load on Unleash instances down to a single call every interval, instead of every single client posting to
-Unleash for updating metrics.
-Unleash instances running on versions older than 4.22 are not able to handle the batch format posted by Edge, which
-means you won't see any metrics from clients connected to an Edge instance until you're able to update to 4.22 or newer.
+Edge is designed to minimize load on its upstream by batching SDK usage metrics. Metrics are gathered over a set interval (`METRICS_INTERVAL_SECONDS`) and sent upstream in a single batch.
 
-## Compatibility
+Unleash versions older than 4.22 cannot process these metrics, so an update is required to see metrics from clients connected to Edge.
 
-Unleash Edge adheres to Semantic Versioning (SemVer) on the API and CLI layers. If you're using Unleash Edge as a
-library in your projects, be cautious, internal codebase changes, which might occur in any version release (including
-minor and patch versions), could potentially break your implementation.
+### Prometheus integration
 
-## Performance
+To monitor the health and performance of your Edge instances, you can consume Prometheus metrics at:
 
-See our page on [Edge benchmarking] [Benchmarking](./docs/benchmarking.md)
+`http://<your-edge-url>/internal-backstage/metrics`
+
+## Compatibility
+
+Unleash Edge adheres to Semantic Versioning (SemVer) on the API and CLI layers. If you're using Unleash Edge as a library in your projects, note that internal changes could affect your implementation, even in minor or patch versions.
 
 ## Debugging
 
-You can adjust the `RUST_LOG` environment variable to see more verbose log output. For example, in order to get logs
-originating directly from Edge but not its dependencies, you can raise the default log level from `error` to `warning`
-and set Edge to `debug`, like this:
+To enable verbose logging, adjust the `RUST_LOG` environment variable. For example, to see logs originating directly from Edge but not its dependencies, you can raise the default log level from `error` to `warning` and set Edge to `debug`, like this:
 
 ```sh
 RUST_LOG="warn,unleash_edge=debug" ./unleash-edge #<command>
 ```
 
-See more about available logging and log levels at https://docs.rs/env_logger/latest/env_logger/#enabling-logging
+See more about available logging and log levels [here](https://docs.rs/env_logger/latest/env_logger/#enabling-logging).
+
+## Additional resources
+
+### Edge concepts
+
+To learn more about Unleash Edge, see the [Concepts](./docs/concepts.md) documentation.
+
+### CLI
+
+For a list of available command-line arguments, see [CLI](/CLI.md).
+
+### Deploying Edge
+
+For deployment instructions, see our [Deploying Edge](./docs/deploying.md) guide.
 
-## Troubleshooting
+### Migrating from Unleash Proxy
 
-### Missing metrics in upstream server
+To migrate from the Unleash Proxy to Unleash Edge, refer to the [migration guide](./migration-guide.md).
 
-#### Possible reasons
```

Comment:
Not sure if we need this, since the availability requirements are covered in the metrics section

Reply:
I think it can come out